src/lights/LightProbe.js

```diff
@@ -26,7 +28,6 @@ LightProbe.prototype = Object.assign( Object.create( Light.prototype ), {
 		Light.prototype.copy.call( this, source );
 
 		this.sh.copy( source.sh );
-		this.intensity = source.intensity;
```

Comment:
`intensity` is already copied in `Light.copy()`.

Reply:
As a FYI, seems like @WestLangley is thinking of removing that inheritance. https://github.com/mrdoob/three.js/pull/18371#issuecomment-603550543

Reply:
Thanks for sharing, good to know! Should I keep the code as it is now?

Reply:
I think so -- for now.